src/watch.ts

```diff
@@ -414,14 +406,13 @@ export function syncTestStatusToVsCode(
   function matchTask(
     task: Task,
     candidates: Set<TestDescribe | TestCase>,
-    type: 'suite' | 'test',
   ): TestDescribe | TestCase {
     let ans: (TestDescribe | TestCase) | undefined
     for (const candidate of candidates) {
-      if (type === 'suite' && !(candidate instanceof TestDescribe))
+      if (task.type === 'suite' && !(candidate instanceof TestDescribe))
         continue
 
-      if (type === 'test' && !(candidate instanceof TestCase))
+      if (task.type === 'test' && !(candidate instanceof TestCase))
```

Comment:
What about type `custom`?

Reply:
I don't understand what's type='custom'. It should be treated as which one? It seems to be newly introduced in v1.